src/index.css

```diff
@@ -35,29 +57,136 @@ ul {
 
 .books-list {
   display: flex;
-  width: 250px;
-  justify-content: space-between;
+  height: 7rem;
+  width: 70%;
+  background-color: #fff;
+  padding: 5px 1.688rem;
+  margin-left: 4.25rem;
+}
+
+.left-block {
+  display: flex;
+  width: 30rem;
+  flex-direction: column;
+  height: 20rem;
+}
+
+.right-block {
+  display: flex;
+  flex-direction: column;
+  padding: 10px 10% 1rem;
 }
 
 #booklist-display {
   display: flex;
+  flex-direction: column;
   gap: 5px;
 }
 
 a {
   text-decoration: none;
-  color: blue;
-  font-size: 20px;
+  color: #121212;
+  opacity: 50%;
+  font-size: 17px;
 }
 
-form {
+#form {
   display: flex;
   flex-direction: column;
   gap: 10px;
+  padding-left: 5.25rem;
+  margin-top: 20px;
+}
+
+form {
+  display: flex;
+  gap: 10px;
+}
+
+input {
+  width: 19.5rem;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 10px;
 }
 
-input,
 .input-submit {
-  height: 2rem;
-  width: 150px;
+  color: #fff;
+  width: 10.5rem;
+  height: 2.713rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #0290ff;
+}
+
+.author {
+  color: #4386bf;
+}
+
+.buttons-div {
+  padding-top: 15px;
+  display: flex;
+  gap: 10px;
+}
+
+.buttons {
+  text-decoration: none;
+  color: #4386bf;
+  border: none;
+  background-color: #fff;
+}
+
+.progress-bar {
+  display: flex;
+  font-size: 12px;
+  justify-content: center;
+  margin: 10px;
+  gap: 10px;
+}
+
+.chapter {
```

Comment:
- [ ] Please implement the element alignment in the progress bar component as it is in the design given. I can better address what I mean by the following screenshot comparisons. Please take a look at them 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/201372802-92ce820d-ebd1-4a64-bad1-d1f02b31bf2d.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/201372914-5b0cc91f-4660-4ce9-8484-5465852cf9bc.png)

I hope you can figure out the difference. As developers, It is important to pay attention to all details even the small ones 😉 👍